data/hlint.yaml

```diff
@@ -1025,7 +1025,7 @@
 # no = x ^^ 18.5
 # instance Arrow (->) where first f = f *** id
 # yes = fromInteger 12 -- 12
-# import Prelude hiding (catch); no = catch
+# import Prelude hiding (print); no = print
```

Comment:
I guess that’s because `catch` is now a keyword?

Reply:
yes exactly.